test/test_meta.py

```diff
@@ -0,0 +1,900 @@
+    torch.ormqr,  # MISSING aten::ormqr
+    torch.pinverse,  # MISSING aten::where.self
+    torch.poisson,  # MISSING aten::poisson
+    torch.polar,  # MISSING aten::polar.out
+    torch.prod,  # MISSING aten::prod
+    torch.qr,  # MISSING aten::_linalg_qr_helper
+    torch.quantize_per_channel,  # MISSING aten::quantize_per_channel
+    torch.quantize_per_tensor,  # MISSING aten::quantize_per_tensor
+    torch.quantize_per_tensor_dynamic,  # MISSING aten::quantize_per_tensor_dynamic
+    torch.relu,  # MISSING aten::relu
+    torch.repeat_interleave,  # MISSING aten::repeat_interleave.Tensor
+    torch.rnn_relu,  # MISSING aten::relu
+    torch.rnn_relu_cell,  # MISSING aten::relu
+    torch.roll,  # MISSING aten::roll
+    torch.rot90,  # MISSING aten::flip
+    torch.rsub,  # MISSING aten::rsub.Tensor
+    torch.searchsorted,  # MISSING aten::searchsorted.Tensor
+    torch.slogdet,  # MISSING aten::linalg_slogdet
+    torch.solve,  # MISSING aten::_solve_helper
+    torch.special.logit,  # MISSING aten::logit
+    torch.special.logsumexp,  # MISSING aten::abs.out
+    torch.special.multigammaln,  # MISSING aten::_local_scalar_dense
+    torch.square,  # MISSING aten::square.out
+    torch.std,  # MISSING aten::std.correction
+    torch.std_mean,  # MISSING aten::std_mean.correction
+    torch.symeig,  # MISSING aten::_symeig_helper
+    torch.take,  # MISSING aten::take
+    torch.threshold,  # MISSING aten::_local_scalar_dense
+    torch.trace,  # MISSING aten::trace
+    torch.var,  # MISSING aten::var.correction
+    torch.var_mean,  # MISSING aten::var_mean.correction
+    torch.vdot,  # MISSING aten::vdot
+    torch.where,  # MISSING aten::where.self
+    torch.quantile,  # MISSING aten::isnan
+    torch.nanquantile,  # MISSING aten::isnan
+}
+
+# Only some overloads/configurations are covered with meta tensors,
+# so we can't use these to toggle expected failure.  Try to prioritize these
+overload_exclude_set = {
+    torch.clamp,  # MISSING aten::clamp.Tensor
+    torch.max,  # MISSING aten::max
+    torch.min,  # MISSING aten::min
+    torch.nn.functional.interpolate,  # MISSING aten::upsample_nearest3d.vec
+    torch.nn.functional.upsample_nearest,  # MISSING aten::upsample_nearest3d.vec
+    torch.nn.functional.pad,  # MISSING aten::reflection_pad2d
+    torch.remainder,  # MISSING aten::remainder.Scalar_Tensor
+    torch.linalg.matrix_rank,  # MISSING aten::linalg_eigh
+    torch.Tensor.isinf,  # MISSING aten::abs.out
+    torch.isinf,  # MISSING aten::abs.out
+    torch.Tensor.isfinite,  # MISSING aten::abs.out
+    torch.isfinite,  # MISSING aten::abs.out
+    torch.diff,  # MISSING aten::logical_xor.out
+}
+
+# These are fine in OpInfo tests, but triggered errors in full test suite
+# crossref testing, which means there is probably not enough coverage from
+# OpInfo.  Patch in https://github.com/pytorch/pytorch/pull/75994 and find
+# out where these fails come from.
+suspicious_exclude_set = {
```

Comment:
nit: not used? do we want to keep it?

Reply:
Yes I do, I'll use it for real crossref testing later